src/swagger/schemas/classes.yml

```diff
@@ -1,94 +1,98 @@
-Class:
+class-model:
+  description: |
+    `Class`
   allOf:
     - $ref: "./combined.yml#/APIReference"
     - type: object
       properties:
         hit_die:
-          description: "The hit die of the class. (ex: 12 == 1d12)."
+          description: "Hit die of the class. (ex: 12 == 1d12)."
```

Comment:
Out of curiosity, when are quotes needed?

Reply:
tldr: they're only needed when something in the string needs to be escaped, or could be confused with a reserved yml indicator

> For most scalars you don’t need any quotes at all, but if you need to define some piece of data which contains characters that could be mistaken with YAML syntax you need to quote it in either double " or single ' quotes for the YAML file to stay valid.
https://docs.octoprint.org/en/master/configuration/yaml.html